batcher/aligned-batcher/Cargo.toml

```diff
@@ -22,7 +22,8 @@ anyhow = "1.0.83"
 ethers = { tag = "v2.0.15-fix-reconnections", features = ["ws", "rustls"], git = "https://github.com/yetanotherco/ethers-rs.git" }
 lambdaworks-crypto = { version = "0.7.0", features = ["serde"] }
 serde_yaml = "0.9.34"
-sp1-sdk = { git = "https://github.com/succinctlabs/sp1.git", rev = "v1.0.8-testnet" }
+sp1-sdk = { git = "https://github.com/succinctlabs/sp1.git", rev = "v1.0.1" }
+axum = "=0.7.4"
```

Comment:
do we actually need this dependency?

Reply:
I was getting dependency conflicts between `sp1-sdk` and another dependency because of the version of `axum` so that's why I pinned it but it seems like the conflicts are no longer happening. Solved in [4ecdbd8](https://github.com/yetanotherco/aligned_layer/pull/670/commits/4ecdbd814b5404e2e0c605fb28c3d11c40320872)